test/development/app-dir/devtool-copy-button/devtool-copy-button.test.ts

```diff
@@ -0,0 +1,20 @@
+import { nextTestSetup } from 'e2e-utils'
+
+describe('app-dir - devtool-copy-button', () => {
+  const { next } = nextTestSetup({
+    files: __dirname,
+    env: {
+      NODE_OPTIONS: '--inspect',
+      DEBUG: '1',
+    },
```

Comment:
not blocking: Should we make it easier to inspect Next.js Server code? E.g. `next dev --inspect`?

Reply:
We've deprecated that flag from `next`, you'll see the error. Guess it's for embracing the general Node.js inspect approach

```
`--inspect` flag is deprecated. Use env variable NODE_OPTIONS instead: NODE_OPTIONS='--inspect' next dev
```